### PR TITLE
clarify a section.

### PR DESCRIPTION
**Pull request by nouiz**

### CODE REVIEW

**Comment by abergeron** [comment]
I agree with the changes, but the docstrings have so many typos that it's making my brain hurt (and I don't mean new typos only).

I should do a cleanup pass to fix the errors, but I don't have the time right now.

Also this need a rebase and you should probably get rid of the merge commit while doing it.

**Comment by nouiz** [comment]
I rebased. I'm not sure what to tell about the typo. Can you check just this PR change and if that part is good, merge it?

Where did you saw typo? Maybe they got fixed by the rebase?

File: doc/lenet.txt [hunk 1]
```diff
@@ -261,10 +266,11 @@ Let's have a little bit of fun with this...
 
         # open random image of dimensions 639x516
         img = Image.open(open('doc/images/3wolfmoon.jpg'))
+        # dimensions are (height, width, channel)
         img = numpy.asarray(img, dtype='float64') / 256.
 
         # put image in 4D tensor of shape (1, 3, height, width)
-        img_ = img.swapaxes(0, 2).swapaxes(1, 2).reshape(1, 3, 639, 516)
+        img_ = img.transpose(2,0,1).reshape(1, 3, 639, 516)
```
Comment (lamblin) [inline doc/lenet.txt]: Spaces after commas for pep8.

**Comment by lamblin** [comment]
Otherwise, I'm fine with it.

File: doc/lenet.txt (hunk 1 above)
Comment (nouiz) [inline doc/lenet.txt]: done.

**Comment by abergeron** [comment]
It does seem like the whole section with docstring changes disappeared in the rebase.

**Comment by lamblin** [comment]
I think the docstring changes were never meant to be part of this PR, there were the ones introduced in other commits that were already merged.

**Pull request merged**